canonical/ubuntu.com
# Apply copy updates

## Done

- Apply copy update as per [doc](https://docs.google.com/document/d/1RBZX3PJPvODxRlcnIS10Ii4SkIMH5Fm74EmsMQ0cnic/edit)

## QA

- View the site locally in your web browser at: http://0.0.0.0:8001/security/esm
- See that the requested changes were made

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-8264
